.github/aur/flux-bin/publish.sh

```diff
@@ -28,20 +28,21 @@ git clone aur@aur.archlinux.org:$PKGNAME $GITDIR 2>&1
 CURRENT_PKGVER=$(cat $GITDIR/.SRCINFO | grep pkgver | awk '{ print $3 }')
 CURRENT_PKGREL=$(cat $GITDIR/.SRCINFO | grep pkgrel | awk '{ print $3 }')
 
-export PKGVER=${VERSION/-/}
+# Transform pre-release to AUR compatible version format
+export PKGVER=${VERSION/-rc./rc}
```

Comment:
So AUR wants `v2.0.0rc1`?

Reply:
That's what I've seen in some examples out there (e.g. `1.2.3alpha1` or `. Other options that would work: are `2.0.0rc.1`, `2.0.0.rc1`, or `2.0.0.rc.1`. I couldn't find any official guidelines, the only thing we know for certain is that `-` is reserved as a separator between the version and the package release number.

Reply:
Hmm that’s strange, none of those are valid semver, according to spec

> A pre-release version MAY be denoted by appending a hyphen

Reply:
I would go with `2.0.0rc.1` which preserves the current replacement `${VERSION/-/}`

Reply:
True, but I don't think Arch intended to fully support or enforce semver in any way. Here's the section about package versioning from the guidelines: https://wiki.archlinux.org/title/Arch_package_guidelines#Package_versioning